Model/Quote/SaveOrderBeforeSalesModelQuoteObserver.php

```diff
@@ -95,38 +100,44 @@ public function execute(\Magento\Framework\Event\Observer $observer)
     }
 
     /**
-     * @param $quote
+     * @param int $quote
      *
      * @return bool
      */
-    private function isMyParcelMethod($quote) {
+    private function isMyParcelMethod(int $quote): ?bool
```

Comment:
deze returned toch altijd een bool, waarom staat `?` erbij

Reply:
En is `$quote` echte een int, uit mijn hoofd zeg ik dat het een string of array is maar ik kan er ook naast zitten